lib/ui/page/home/page/chat/widget/message_info/view.dart

```diff
@@ -219,4 +145,108 @@ class MessageInfo extends StatelessWidget {
       },
     );
   }
+
+  /// Represents a table row for table with info
+  TableRow _tableRow(
+    Style style,
+    String label,
+    Widget child, {
+    double addPadding = 0,
+  }) => TableRow(
+    children: [
+      Padding(
+        padding: EdgeInsets.only(top: 4 + addPadding),
+        child: Text(
+          label,
+          style: style.fonts.small.regular.secondary,
+          textAlign: TextAlign.right,
+        ),
+      ),
+      Padding(
+        padding: const EdgeInsets.fromLTRB(16, 4, 4, 0),
+        child: Align(
+          alignment: Alignment.centerLeft,
+          child: DefaultTextStyle(
+            style: style.fonts.small.regular.secondary.copyWith(
+              color: style.colors.secondaryBackgroundLight,
+            ),
+            child: child,
+          ),
+        ),
+      ),
+    ],
+  );
+
+  /// Returns localized string of [chatItem] status
+  String _getLabelStatus() {
+    if (reads.isNotEmpty) {
+      return 'label_message_status_read'.l10n;
+    }
+
+    if (chatItem!.status.value.name == 'sent') {
+      return 'label_message_status_delivered'.l10n;
+    }
+
+    if (chatItem!.status.value.name == 'sending') {
+      return 'label_message_status_sent'.l10n;
+    }
+
+    return 'label_message_status_not_sent'.l10n;
+  }
+}
+
+class _UsersListWidget extends StatelessWidget {
+  const _UsersListWidget({required this.c, required this.reads});
+
+  final MessageInfoController c;
+  final Iterable<LastChatRead> reads;
+
```

Comment:
1) Documentation.
2) Since this widget accepts `MessageInfoController`, then it's more of a component than a widget, I guess. But that's ok to be a private widget, maybe? Not sure.